.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 🐛 Bug Report
+about: If something isn't documented correctly 🤔
+
```

Comment:
Add a hint that it is preferred to submit a suggestion via PR ("Edit on GitHub"), if possible?

Reply:
Done